### PR TITLE
RESTful URL fixes.

### PR DESCRIPTION
**Pull request by dkorolev**
Hi @nyospe -- thanks for pinpointing the issue.

Part of it was indeed taken care of by the nginx fix (oops), and the remaining logic is being corrected by this PR.

CC @mzhurovich

### CODE REVIEW

File: Storage/rest/structured.h [hunk 1]
```diff
@@ -154,8 +154,10 @@ struct Structured {
       } else {
         if (!input.export_requested) {
           // Top-level field view, identical for dictionaries and matrices.
+          // Pass `url` twice, as `pagination_url` and `collection_url` are the same for this format.
+          const std::string url = input.restful_url_prefix + '/' + kRESTfulDataURLComponent + '/' + input.field_name;
           return RESPONSE_FORMATTER::template BuildResponseWithCollection<PARTICULAR_FIELD, ENTRY, ENTRY>(
-              context, input.restful_url_prefix + '/' + kRESTfulDataURLComponent + '/' + input.field_name, input.field);
+              context, url, url, input.field);
```
Comment (nyospe) [inline Storage/rest/structured.h]: :heavy_check_mark:

File: Storage/rest/structured.h [hunk 2]
```diff
@@ -207,6 +214,8 @@ struct Structured {
             context,
             input.restful_url_prefix + '/' + kRESTfulDataURLComponent + '/' + input.field_name + '.' +
                 MatrixContainerProxy<KEY_COMPLETENESS>::PartialKeySuffix(),
+            input.restful_url_prefix + '/' + kRESTfulDataURLComponent + '/' + input.field_name + '.' +
```
Comment (nyospe) [inline Storage/rest/structured.h]: Bleah. But, it does work. Might be good to add a comment to explain the redundancy.

**Comment by nyospe** [comment]
LGTM

File: Storage/rest/structured.h (hunk 2 above)
Comment (dkorolev) [inline Storage/rest/structured.h]: Good catch, https://github.com/C5T/Current/pull/646.

**Pull request merged**